src/openrct2-ui/windows/Ride.cpp

```diff
@@ -1598,28 +1598,30 @@ static void window_ride_init_viewport(rct_window* w)
             focus.sprite.type |= VIEWPORT_FOCUS_TYPE_SPRITE;
         }
     }
-    else if (eax >= ride->num_vehicles && eax < (ride->num_vehicles + ride->num_stations))
+    else if (viewSelectionIndex >= ride->num_vehicles && viewSelectionIndex < (ride->num_vehicles + ride->num_stations))
     {
-        StationIndex stationIndex = STATION_INDEX_NULL;
-        int32_t count = eax - ride->num_vehicles;
-        do
+        int32_t count = viewSelectionIndex - ride->num_vehicles;
+        
```

Comment:
clang format isn't happy with the whitespace here i think you have a number of spaces either delete the spaces or remove the line completely to satisfy clang format (or even better run clang format locally its built into Visual Studio if that is what you are using)